### ISSUE TITLE
Local API error when importing youtube subscription

### ISSUE THREAD

**Issue by jgsource** [FreeTubeApp/FreeTube#1848]
When I import the subscription.csv file, I get: "Local API error This account has been terminated for violating YouTube's Community Guidelines. Falling back to invidous API." My subscriptions are also still empty after this. Any way I can fix this? I am on linux with specifically the AUR freetube-bin.

**Comment by harl** [comment]
I'm in the same situation here.
I downloaded my latest YouTube export from Google Takeout, which contains a CSV file with my subscriptions.
Both the local API as well as the Invidious API have hindered importing my subscriptions.
I wouldn't need them to be refreshed right away, at least not immediately when they're imported.
There's no reason for each and every subscription to be readily loaded before I actively visit them, anyway.

But they're not just not refreshed, they're not even imported as basic links to the respective channels.
I'm subscribed to about 900 channels, and those APIs seem to have already hit their limits even before I tried.
Please let us import subscriptions without all the fancy stuff that queries additional data.

**Comment by ChunkyProgrammer** [comment]
Once #1783 gets merged, adding the ability to not do those additional queries would be possible. If i remember correctly, the import does all the web requests just to get the channel's thumbnail. (Which can be set once you navigate to a video by that channel or the channel itself instead of when you are importing)

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
is any of u still facing this issue?

**Comment by jmcelroy01** [comment]
> is any of u still facing this issue?

Yes. It gives me local API error status code 429 repeatedly as the import runs.

**Comment by ChunkyProgrammer** [comment]
Once https://github.com/FreeTubeApp/FreeTube/pull/2604 gets merged, I'll open a pr to add a setting to disable fetching thumbnails on initial import which should fix the issues with 429 error :)
